android/pytorch_android/CMakeLists.txt

```diff
@@ -122,19 +122,37 @@ if(ANDROID_ABI)
       libclog
   )
 else()
+  find_package(nnpack)
+  find_package(XNNPACK)
+  find_package(pthreadpool)
+  find_package(pytorch_qnnpack)
+
   # Prefer dynamic linking on the host
   set(pytorch_jni_LIBS
       fbjni
       torch
       torch_cpu
       c10
-      nnpack
-      XNNPACK
-      pytorch_qnnpack
-      pthreadpool
       cpuinfo
       clog
   )
+
+  if(nnpack_FOUND)
```

Comment:
Here (and in the other libs), you're making the libraries optional on non-windows platforms as well.
The main CMakeLists.txt seems to use a USE_NNPACK option instead. If this difference intentional?

Reply:
Not intentional - just a lack of experience with cmake. 😄  It seems like I should just be able to replace the `find_package` and `if(*_FOUND)` with appropriate `if(USE_*)` checks - I can fix that up.

Reply:
I'll liberally admit that I wouldn't know if it is a problem or not, so an alternative strategy to addressing my drive-by comment would be to see if someone who actually knows something (@dreiss or @ljk53 or one of the other cool people) have something to say.

Reply:
I will commit the change with the option to revert as needed. 😄

Reply:
See e62256e2289

Reply:
The failure looks like a one-off docker thing - it _seems_ unrelated to the fix. I rebased and that may have caused something funny to happen.